src/Events/Migration/v0.33/01-alter-function.sql

```diff
@@ -0,0 +1,47 @@
+DROP FUNCTION IF EXISTS events.getappevents(character varying, character varying, timestamp with time zone, timestamp with time zone, text[], text[], integer);
+CREATE OR REPLACE FUNCTION events.getappevents(
+	_subject character varying,
+	_after character varying,
+	_from timestamp with time zone,
+	_to timestamp with time zone,
+	_type text[],
+	_source text[],
+	_size integer)
+    RETURNS TABLE(cloudevents text)
+    LANGUAGE 'plpgsql'
+    --COST 100
+    --VOLATILE PARALLEL UNSAFE
```

Comment:
Was the intention of commenting out these two lines?

Reply:
Forgot to remove them. Just default values.